games.py

```diff
@@ -41,6 +41,9 @@ def min_value(state):
 
 # ______________________________________________________________________________
 
+dice_rolls = list(itertools.combinations_with_replacement([1, 2, 3, 4, 5, 6], 2))
+direction = {'W' : -1, 'B' : 1}
+
 def expectiminimax(state, game):
```

Comment:
I know this is not your fault, somehow the code got like this before, but `expectiminimax` **cannot** be limited to games with two dice. It should be a line-for-line implementation of the pseudocode:

      EXPECTIMINIMAX(s) =
         UTILITY(s)                        if TERMINAL-TEST(s)
        max_a EXPECTIMINIMAX(RESULT(s, a)) if PLAYER(s)= MAX
        min_a EXPECTIMINIMAX(RESULT(s, a)) if PLAYER(s)= MIN
       ∑_r P(r)EXPECTIMINIMAX(RESULT(s, r)) if PLAYER(s)= CHANCE